en-us/faq.md

```diff
@@ -0,0 +1,29 @@
+# Vanilla Launcher FAQ
+
+### How to install?
+
+- There are many ways to install FO on multiple launchers. Please use this [guide](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions#minecraft-launcher-vanilla) to install FO on the launcher that you use.
+
+### So I just copy the mods, right?
+
+- No. Fabulously Optimized consists of mods, configs and a resource pack that is built to help the user naviagte through Mod Menu. If you simply copy the mods over, the settings would not be applied, and since you would have to configure the mods yourself, you might not have the proper settings to accurately replicate OptiFine.
+
+### What is the MultiMC ZIP?
+
+- MultiMC ZIP is a packaged instance for the launcher called MultiMC. It is distributed in CurseForge because MultiMC itself no longer supports downloading CurseForge Modpacks and downloads from the ZIP file support the creator of FO. This also allows for a better installation of FO for vanilla launcher users.
+
+### What is in the ZIP file?
+
+- In both the CurseForge and MultiMC ZIP files, they contain configs for each mod and resource packs which are essential for FO to run. Without these files, the modpack would not be configured properly which could lead to not having the right settings to replicate OptiFine.
+
+### Why try a different launcher?
+
+- There are a variety of launchers available, each with unique features. If you use the Vanilla Launcher, you miss out on many features, including the ability to update the modpack and the version of Minecraft, add any mods with a few simple clicks, and finally download the modpack more easily.
+
+### Why is FO not a client?
+
+- FO is not a client because simply we would need to be able to code a client which would require people to know the programming language that we are writing the client in. Also, we can't bundle a bunch of mods without thinking about the license that they are using. Some licenses don't allow for distribution or for commercial use which could affect some mods being excluded from the client.
+
+### Is there an easier way to install FO?
+
+- For now, we recommend using other launchers like CurseForge Launcher, MultiMC and GDLauncher as provide an easy way to install the modpack as well as to upgrade it. However, if you want to install FO on the Vanilla Launcher, you can follow this [guide](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions#minecraft-launcher-vanilla) until we have a better solution. Soon, we are going to release a Vanilla Installer which provides an easier way to install the modpack on the Vanilla Launcher.
```

Comment:
> Soon, we are going to release a Vanilla Installer which provides an easier way to install the modpack on the Vanilla Launcher.

again with the capitalization, maybe link to the repo (https://github.com/Fabulously-Optimized/vanilla-installer)
Hoping we can get a beta out soon-ish, but not sure yet.